### PR TITLE
Sagersweb bio

### PR DESCRIPTION
**Pull request by sagersweb**
Code complete(ish)
- [x] Awesome?
- [x] Rad?

### CODE REVIEW

File: sagersweb-bio.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#Sagersweb Bio
+
+* Voted "most awesome dad" by over 4 kids
+* Won school science fair 7 years in a row
+* Enjoys a good sandwich
+* Moonlights as a graphic designer
+* Geoffrey, Not Jeff, Jeffry, or Gregory.
```
Comment (geothird) [inline sagersweb-bio.md]: Can you pronounce Geoffrey?

Comment (sagersweb) [inline sagersweb-bio.md]: @geothird  - http://www.oxfordlearnersdictionaries.com/us/definition/english/geoffrey-chaucer?q=Geoffrey

**Comment by sagersweb** [comment]
@katymdc - the total is 5, but 5 IS over 4, so there's that.

**Comment by sagersweb** [comment]
@johnsonralan I graduated in Business Information Systems, the graphic design is a hobby.

**Pull request merged**